ui/ozone/platform/wayland/wayland_window.cc

```diff
@@ -203,6 +203,7 @@ void WaylandWindow::Minimize() {
   // here. We can track if the window was unminimized once wayland sends the
   // window is activated, and the previous state was minimized.
   state_ = PlatformWindowState::PLATFORM_WINDOW_STATE_MINIMIZED;
+  delegate_->OnWindowStateChanged(state_);
 }
```

Comment:
Even if it's not a change from this CL, 'WaylandWindow::Minimize()' has double 'DCHECK(xdg_surface_);'. Could you remove one?